reconcile/test/glitchtip/test_glitchtip.py

```diff
@@ -26,8 +31,10 @@ def sort_all(orgs: list[Organization]) -> list[Organization]:
 
 
 def test_fetch_current_state(
-    glitchtip_client: GlitchtipClient, glitchtip_server_full_api_response, fx
-):
+    glitchtip_client: GlitchtipClient,
+    glitchtip_server_full_api_response: Optional[None],
```

Comment:
This one looks odd :thinking:

Reply:
I've updated it to `None` only, nevertheless the pytest fixture `glitchtip_server_full_api_response` returns `None`, so it's correct :)